internal/util-complete/src/test/scala/ParserTest.scala

```diff
@@ -21,6 +21,7 @@ object JLineTest {
   val five = (num ~ token("+" | "-") ~ num) <~ token('=') flatMap {
     case a ~ "+" ~ b => token((a + b).toString)
     case a ~ "-" ~ b => token((a - b).toString)
+    case _           => failure("Unexpected pattern")
```

Comment:
Needed to slience

```
[error] -- [E029] Pattern Match Exhaustivity Error: /home/runner/work/sbt/sbt/internal/util-complete/src/test/scala/ParserTest.scala:22:4 
[error] 22 |    case a ~ "+" ~ b => token((a + b).toString)
[error]    |    ^
[error]    |    match may not be exhaustive.
[error]    |
[error]    |    It would fail on pattern case: ((_, _), _)
```